Assets/Scenes/navmesh fixedplayer.unity

```diff
@@ -0,0 +1,9758 @@
+%YAML 1.1
```

Comment:
Is this scene supposed to be included?

Reply:
ill delete it @ValgardurRI  is done with it.